app/code/Magento/Checkout/view/frontend/web/js/model/checkout-data-resolver.js

```diff
@@ -219,15 +219,27 @@ define([
          */
         applyBillingAddress: function () {
             var shippingAddress;
+            var isBillingAddressInitialized;
 
             if (quote.billingAddress()) {
                 selectBillingAddress(quote.billingAddress());
 
                 return;
             }
-            shippingAddress = quote.shippingAddress();
 
-            if (shippingAddress &&
+            if(quote.isVirtual()) {
```

Comment:
Hi @LucasCalazans,
Please add space between `if`and `(`.

Reply:
Hi @williankeller, thanks for your review, I've just added this alteration.